### ISSUE TITLE
upright integrals

### ISSUE THREAD

**Issue by kia999** [firamath/firamath#49]
It seems that upright integrals are not scaled up as ordinary integrals.

```
    \setmathfont[Color=000066,Scale=MatchLowercase]{Fira Math}
    \setmathfont[range={"222B-"2233,"2A0B-"2A1C},StylisticSet=1]{Fira Math}

    \begin{equation}
    \label{2:06}
    n(r)
    =
    2  \int_{0}^{\infty } \dif{v} v
    \int_{-\arcsin(r/a)}^{+\arcsin(r/a)}
    \overline{f}(v,\alpha; r)\dif\alpha
    .
    \end{equation}

```

**Comment by stone-zeng** [comment]
Are you using XeLaTeX or LuaLaTeX? I can only see this problem with XeLaTeX.

**Comment by kia999** [comment]
I am using xeltatex.

Igor A. Kotelnikov
igor.kotelnikov@gmail.com

Budker Institute of Nuclear Physics
Novosibirsk, 630090, Russia
tel: +7 383 3294268 (office)
mobile: +7 913 933 24 84
(also WhatsApp, WeChat)



пн, 13 янв. 2020 г. в 00:50, Xiangdong Zeng <notifications@github.com>:

**Comment by stone-zeng** [comment]
I will update the CTAN version as soon as possible.

**Comment by kia999** [comment]
Dear  Xiangdong Zeng,

I updated firamth yesterday, January 15, with miktex update package of
firamath dated also by January 15, but the problem with upright integrals
in xelatex remained. Lualatex produces correct result.

Best regards,


Igor A. Kotelnikov
igor.kotelnikov@gmail.com

Budker Institute of Nuclear Physics
Novosibirsk, 630090, Russia
tel: +7 383 3294268 (office)
mobile: +7 913 933 24 84
(also WhatsApp, WeChat)



пн, 13 янв. 2020 г. в 16:25, Xiangdong Zeng <notifications@github.com>:

**Comment by stone-zeng** [comment]
Can you locate the font file (in `.../texmf-dist/fonts/opentype/public/firamath/`) and check its version (right click then select "Properties" or something similar)? In both my and my friend's computer, the problem has been fixed in the lastet **v0.3.3** version.

And you can directly load font by its filename (`\setmathfont{FiraMath-Regular.otf}[Path=...]`) then check whether the problem still remains.

**Comment by kia999** [comment]
Version was 0.3.3. I uninstalled firamath-otf package, but
firamth-regular.oft remained in the font folder of MilTeX, reloaded
computer. manually remove firamath-regular.otf, ran xelatex and got error
because of absent font. Then I reinstalled firamath package and now have
correct upright integrals both with xelatex and lualatex.

Thank you.

Igor A. Kotelnikov
igor.kotelnikov@gmail.com

Budker Institute of Nuclear Physics
Novosibirsk, 630090, Russia
tel: +7 383 3294268 (office)
mobile: +7 913 933 24 84
(also WhatsApp, WeChat)



чт, 16 янв. 2020 г. в 15:05, Xiangdong Zeng <notifications@github.com>:
